generate/generate.go

```diff
@@ -720,30 +709,19 @@ func (g *Generator) convertSecurityDefinitions(lit *ast.CompositeLit) (*options.
 			for _, v := range ss.(*ast.KeyValueExpr).Value.(*ast.CompositeLit).Elts {
 				n := v.(*ast.KeyValueExpr)
 				switch key := n.Key.(*ast.Ident).Name; key {
-				case "Type":
-					sel, ok := options.SecurityScheme_Type_value[n.Value.(*ast.SelectorExpr).Sel.Name]
+				case "Type", "In", "Flow":
+					enumMap := proto.EnumValueMap(prefix + key)
```

Comment:
I experimented with a way to not hard-code the `Foo_Type_value` maps, and it kinda works, but requires me to have the prefix hard-coded. Not sure if there's a cleaner way to do it.